### PR TITLE
qemu_vm: fix `status_test_command` error

### PR DESCRIPTION
**Pull request by sitoliu**
ID: 1685046
Signed-off-by: Sitong Liu <siliu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@sitoliu Please describe the issue in the commit log, thanks.

**Comment by vivianQizhu** [comment]
@sitoliu Bug id please, any issue you hit or is there any actual usage of it?

**Comment by sitoliu** [comment]
@vivianQizhu Created a bz for it, please help review it again, thanks.

**Comment by vivianQizhu** [comment]
I would recommend to demonstrate in the commit log that it is required here because windows has a different `status_test_command` defined in `params`. Thanks.

**Comment by vivianQizhu** [comment]
@pezhang

**Comment by pezhang** [comment]
Looks good to me.
Acked by: Pei Zhang pezhang@redhat.com

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -2458,7 +2458,8 @@ def _create_serial_console(self):
             auto_close=False,
             output_func=utils_misc.log_line,
             output_params=(log_name,),
-            prompt=self.params.get("shell_prompt", "[\#\$]"))
+            prompt=self.params.get("shell_prompt", "[\#\$]"),
+            status_test_command=self.params.get("status_test_command", "echo $?"))
```
Comment (vivianQizhu) [inline virttest/qemu_vm.py]: This line is too long.

**Pull request merged**